modules/Fetch.js

```diff
@@ -166,6 +177,7 @@ class Fetch extends Component<Props> {
         method,
         onTimeout,
         onProgress,
+        onIntercept: interceptor,
```

Comment:
You just have to write: `onIntercept: context.rdfInterceptor || onIntercept` here and that should do the trick!

Reply:
I think it should be `onIntercept || context.rdfInterceptor` since we want `<Fetch>'s onIntercept` to overide `<ConnectedFetch> onIntercept`